sample/src/main/java/no/finntech/rxactivityresponse/sample/RxLoginExample.java

```diff
@@ -0,0 +1,182 @@
+                                          Account account = new Account(accountName, GoogleAuthUtil.GOOGLE_ACCOUNT_TYPE);
+                                          String scopes = "oauth2:" + GOOGLE_PLUS_SCOPES;
+                                          try {
+                                              subscriber.onNext(GoogleAuthUtil.getToken(activity.getApplicationContext(), account, scopes));
+                                              subscriber.onCompleted();
+                                          } catch (IOException e) {
+                                              subscriber.onError(e);
+                                          } catch (final GoogleAuthException e) {
+                                              if (e instanceof UserRecoverableAuthException) {
+                                                  handler.post(new Runnable() {
+                                                      @Override
+                                                      public void run() {
+                                                          RxActivityResponseDelegate rxActivityResponseDelegate = RxActivityResponseDelegate.get(activity);
+                                                          if (!rxActivityResponseDelegate.hasActiveResponse()) {
+                                                              rxActivityResponseDelegate.setResponse(responseHandler);
+                                                              activity.startActivityForResult(((UserRecoverableAuthException) e).getIntent(), rxActivityResponseDelegate.getRequestCode());
+                                                          }
+                                                      }
+                                                  });
+                                                  subscriber.onCompleted();
+                                                  return;
+                                              }
+                                              subscriber.onError(e);
+                                          }
+                                      }
+                                  };
+                                  subscriber.add(s);
+                                  return s;
+                              }
+                          }
+                    ).observeOn(AndroidSchedulers.mainThread());
+        }
+    }
+
+    public void getLoginToken() {
+        final Activity activity = (Activity) getContext();
+        String[] permissions = new String[]{Manifest.permission.GET_ACCOUNTS};
+
+        final Scope[] scopes = {new Scope(Scopes.PROFILE), new Scope(Scopes.EMAIL)};
+        SnackbarRationaleOperator rationaleOperator = new SnackbarRationaleOperator(this, "Need permission for ...");
+
+        /*RxPlayServices.getPlayServices((Activity) getContext(), responseHandler, permissions, scopes, Plus.API).subscribe(new Action1<GoogleApiClient>() {
```

Comment:
Leaving it commented in but added the comment:
Example code to revoke play services to reset the flow (play services doesn't ask again once an app has been given access. You dont want this in production, but it's handy for testing